packages/lib/person/service.ts

```diff
@@ -106,7 +110,8 @@ export const getPersonCached = async (personId: string) =>
   )();
 
 export const getPeople = cache(async (environmentId: string, page: number = 1): Promise<TPerson[]> => {
```

Comment:
Can we find a generic approach for pagination?
For all the other services we currently return all records when the `get[Records]` method is called. Does it make sense to here make `number` optional and if set return only the page, if not set it returns all records?
that way it's working with the current pattern. What do you think?

Reply:
Had to think about this and then fix other concerns before responding to this.

Yeah that's a good idea. If we standardize the pagination, we will need to expose a constant like we have for people `PEOPLE_PER_PAGE` to have more like an `ITEMS_PER_PAGE` or `ROWS_PER_PAGE`. Then for every `findMany` calls we optionally add pagination.

Is that correct?

Reply:
yes :-) 
I would like to get all results by default and pagination only when a page is provided; by having it like this

```ts
export const getPeople = cache(async (environmentId: string, page?: number): Promise<TPerson[]> => {
```

I think changing `PEOPLE_PER_PAGE` to `ITEMS_PER_PAGE` is a good idea :-) 

Then the approach can later be copied for more listing services and we can also build it into the APIs